src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int COUNT_OF_FIGURE = 6;
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSuplier = new FigureSupplier();
```

Comment:
The variable name `figureSuplier` contains a typo; it should be `figureSupplier`. Correct spelling is important for code readability and maintainability.